kili/mutations/user/__init__.py

```diff
@@ -28,33 +28,60 @@ def __init__(self, auth):
         """
         self.auth = auth
 
+    @deprecate(
+        """
+        The parameter "name" is deprecated since: 11/01/2022.
+        It will be removed after: 21/02/2022.
+        Parameters "firstname" and "lastname" have to be used instead.
+            > kili.create_user(
+                firstname='John',
+                lastname='Miller',
+                email='john.miller@kili-technology.com',
+                password='Kili123',
+                organization_role='ADMIN')
+        """)
     @Compatible(['v1', 'v2'])
     @typechecked
-    def create_user(self, name: str, email: str, password: str, organization_role: str):
+    def create_user(self,
+                    email: str,
```

Comment:
You can not change the order of the props, else it is breaking !

Reply:
I understand but there is would lead to a non-default argument following a default one (as name is now optional with default none)...

Reply:
- Keep the order of the arguments
- When name is not `None`, emit a deprecation warning

Reply:
Did you warn Antoine de Séveracnabotu this change?